packages/airnode-node/src/constants.test.ts

```diff
@@ -23,5 +23,6 @@ describe('constants', () => {
     expect(constants.PRIORITY_FEE).toEqual(3120000000);
     expect(constants.BASE_FEE_MULTIPLIER).toEqual(2);
     expect(constants.MAXIMUM_ONCHAIN_ERROR_LENGTH).toEqual(100);
+    expect(constants.PROCESSING_TIMEOUT).toEqual(10_000);
```

Comment:
Btw. your solution handles timeouts well, but I think it would be nice to add a test which tests that. Something like:

```js
const preProcessingSpecifications = [
        {
          environment: 'Node 14' as const,
          value: 'const output = {...input, from: `ETH`}; while(true);',
          timeoutMs: 5_000,
        },
        ...
```